synfig-studio/src/gui/dialogs/dialog_guide.cpp

```diff
@@ -96,8 +98,35 @@ Dialog_Guide::Dialog_Guide(Gtk::Window& parent, etl::handle<synfig::Canvas> canv
 	guideGrid->attach(*angle_widget      , 1, 0, 1, 1);
 	guideGrid->attach(angle_type_picker  , 2, 0, 1, 1);
 
+	Gtk::Frame* pivotFrame = manage(new Gtk::Frame(_("Set Pivot Position")));
+	pivotFrame->set_shadow_type(Gtk::SHADOW_NONE);
+	(static_cast<Gtk::Label*>(pivotFrame->get_label_widget()))->set_markup(_("<b>Set Pivot Position</b>"));
+	pivotFrame->set_margin_bottom(5);
+	pivotFrame->set_margin_top(5);
+	pivotFrame->set_margin_left(5);
+
+	auto posGrid = manage(new Gtk::Grid());
+	posGrid->get_style_context()->add_class("dialog-secondary-content");
+	posGrid->set_row_spacing(6);
+	posGrid->set_column_spacing(8);
+	
+
+	Gtk::Label* xPosLabel = manage(new Gtk::Label(_("_X:"), true));
+	Gtk::Label* yPosLabel = manage(new Gtk::Label(_("_Y:"), true));
+	x_widget=manage(new Gtk::SpinButton(x_adjustment,15,2));
+	x_widget->show();
+	y_widget=manage(new Gtk::SpinButton(y_adjustment,15,2));
```

Comment:
Please use spaces surrounding the equal sign, like you did for `Gtk::Label` vars.